website/docs/recipes/munit.md

```diff
@@ -0,0 +1,115 @@
+		+ pactServicePort 
+		+ " -l " 
+		+ mockPactServerPort 
+		+ " -u " 
+		+ mockPactServerPort
+		+ " --broker "
+		+ brokerUrl
+		+ " -v 3"; 
+		
+	Process pactService = Runtime.getRuntime().exec(command);
+		
+	return pactService;
+}
+	
+public static void stopPactService() throws ClientProtocolException, IOException {
+	pactService.destroy();
+}
+```
+
+By setting the service to a variable, we will be able to then stop the running process later with the `.destroy()` command.
+
+More information on the Pact JVM can be found here: https://docs.pact.io/implementation_guides/jvm/pact-jvm-server
+
+### From Anypoint Studio
+
+From Anypoint Studio, we will use the "Invoke Static" Java block to call the `startPactService()` method within the PactService class. This will start running the Pact standalone server as a separate Java process on the machine. 
+
+![Java Invoke static widget](../../static/img/docs/recipes/munit/start-service.png) 
+
+## Creating a Mock Service
+
+Now that we have the service running, all we need to do is make some API calls to localhost with the port. We can use the Java "Invoke static" mule widget for this as we only need to make API calls to set up the Mock API Provider. Here is an example of using a CloseableHttpClient but you can use any preferred method for making the API calls from Java.  
+
+```Java
+public static void createPactMockServer(String pactBody) throws ClientProtocolException, IOException, InterruptedException {
+  CloseableHttpClient client = HttpClients.createDefault();
+  HttpPost post = new HttpPost("http://localhost:" + pactServicePort + "/create?state=NoUsers&path=/sub/ref/path");
+  ResponseHandler<String> handler = new BasicResponseHandler();
+		
+  post.setHeader("Content-Type", "application/json");
+  post.setEntity(new StringEntity(pactBody));
+		
+  client.execute(post, handler);
+	    	    
+  close();
+}
+```
+
+The variable pactServicePort is the port that the Pact service is running on. The pactBody can be passed from another Java function that can use the `ConsumerPactBuilder` to set up the pact interactions that would be required to be sent. This would be very similar to the example logic provided here: https://docs.pact.io/implementation_guides/jvm/consumer
+
+When setting up the `pactBody` variable, I found the need to use the "New" and "Invoke" Mule Java blocks in order to avoid errors. However, there might be other solutions where static methods can be used like the rest of the solution. 
+
+### From Anypoint Studio
+
+An example setup from Anypoint Studio would be like the below image passing a variable from another Java class.
+
+![Example of creating the Mock Server](../../static/img/docs/recipes/munit/create-mock-server.png) 
+
+### Hooking up to MUnit test
+
+Finally, to connect to the service, all you need to do is set the API URL to localhost. This can be done through environment variables or other techniques that you would use for writing MUnit tests normally. 
+
+# Provider
```

Comment:
I think this should be ## as otherwise its being swallowed in the right hand side bar. You can't have more than one H1 markdown heading in docusauraus

you may need to bring in the others that are ## to ###

assume you wanted consumer and provider to appear at top level headings in the side bar?

<img width="357" alt="" src="https://github.com/pact-foundation/docs.pact.io/assets/19932401/aad57f10-880a-4ebe-9d1d-87c967443660">